src/System.EventLog/tests/System.EventLog.Tests.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
+  <PropertyGroup>
+    <ProjectGuid>{5B218883-369E-4C3D-8BD0-74843474DCBD}</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="EventLogTests/EventLogTests.cs" />
+    <Compile Include="EventLogTests/EventLogTestsBase.cs" />
+    <Compile Include="EventLogTests\Class1.cs" />
```

Comment:
You seem to have not included this file, hence compile error

Reply:
I removed it

Reply:
ok